src/rlookup.c

```diff
@@ -855,7 +870,7 @@ static int loadIndividualKeys(RLookup *it, RLookupRow *dst, RLookupLoadOptions *
       if (!(kk->flags & RLOOKUP_F_SCHEMASRC)) {
         continue;
       }
-      if (!options->noSortables) {
+      if (!options->forceLoad) {
```

Comment:
check for dead code, (either here or on other PR)

Reply:
will be checked on another.